dev/dags/basic_cosmos_dag.py

```diff
@@ -18,7 +18,7 @@
     profile_mapping=PostgresUserPasswordProfileMapping(
         conn_id="airflow_db",
         profile_args={"schema": "public"},
-        disable_event_tracking=True,
+        dbt_config_vars=DbtProfileConfigVars(send_anonymous_usage_stats=True),
```

Comment:
`disable_event_tracking` should be kept here so that issues like that which was fixed in #784 are caught in the integration tests.

Maybe a different field can be set in `dbt_config_vars` here?

Reply:
fix